src/components/trends/smbg/FocusedSMBGPointLabel.js

```diff
@@ -57,7 +65,11 @@ const FocusedSMBGPointLabel = (props) => {
   const pointTooltips = _.map(allSmbgsOnDate, (smbg, i) => (
     <Tooltip
       key={i}
-      content={<span className={styles.number}>{displayBgValue(smbg.value, bgUnits)}</span>}
+      content={
+        <span className={styles.number}>
+          {displayBgValue(smbg.value, bgUnits, getOutOfRangeThreshold(smbg))}
```

Comment:
You make a decent point and I think it's probably good to keep it in mind going forward (if we find ourselves desiring a differentiation between the formatting functions more often), but since this isn't common yet and is executed a relatively few number of times, I think it's fine to just roll with this solution for now.